### PR TITLE
consume extracted paramfetch + basic CircleCI config

### PR DESCRIPTION
**Pull request by laser**
## What's in this PR?

We've recently extracted the paramfetch software into a new repo (go-paramfetch). This changeset updates go-sectorbuilder to consume that newly-extracted code. It also runs the sectorbuilder tests in a CircleCI build container on each push to GitHub.

### CODE REVIEW

File: sectorbuilder_test.go [hunk 1]
```diff
@@ -195,9 +204,6 @@ func TestSealAndVerify(t *testing.T) {
 }
 
 func TestSealPoStNoCommit(t *testing.T) {
-	if testing.Short() {
```
Comment (acruikshank) [inline sectorbuilder_test.go:198]: Removing this and the one below seems unrelated to this PR. Was this intentional?

**Review by acruikshank — APPROVED** [review]
This looks good. I'm not sure why some of the long running tests are no longer skipped in short mode. If you have a good reason, this is good to go.

File: sectorbuilder_test.go (hunk 1 above)
Comment (laser) [reply sectorbuilder_test.go:198]: > Was this intentional?

There aren't any tests in this Git repo which qualify as "short," so IMHO there's no utility in that check.
